content/docs/user-guide/visualization/animation/animation-editor/ragdoll/_index.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Ragdoll
+date: 2022-09-02
+---
+
+A ragdoll is a physical representation of a character in the animation system that you can use to simulate behavior, such as hit reactions and character death. The physical representation consists of a hierarchy of rigid bodies with simple shapes that are connected by joints. The animation system and the PhysX system work together to simulate the realistic behaviors. While the ragdoll setup occurs in the animation system, the PhysX system is responsible for how a character moves based on environmental interactions and external forces. For example, you can set up your ragdoll so that the character will rotate as it collapses when you apply force to the character's outer shoulder area.
```

Comment:
suggestion: 'simulate the realistic behaviors' -> 'simulate realistic behaviors'

Reply:
suggestion: 'when you apply force' -> 'when you apply a force'

Reply:
Fixed.